scripts/sign_rproc_fw.py

```diff
@@ -0,0 +1,511 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (C) 2023, STMicroelectronics - All Rights Reserved
```

Comment:
Do you need the "All Rights Reserved" clause? It's rather pointless, but OK if required by your legal team.

Reply:
I removed it